src/dbseeder/__main__.py

```diff
@@ -37,6 +38,8 @@ def main():
         return seeder.create_tables(who=params['who'])
     elif arguments['postprocess']:
         return seeder.post_process(who=params['who'])
+    elif arguments['update']:
+        return seeder.update(source=arguments['<source>'], who=arguments['<configuration>'])
```

Comment:
I would propose that we get rid of `params` (line 26) above and call all of the functions in this manner for consistency.